test/test_torch.py

```diff
@@ -2549,6 +2549,14 @@ def test_randn(self):
         torch.randn(SIZE, SIZE, out=res2)
         self.assertEqual(res1, res2)
 
+    def test_randint(self):
+        torch.manual_seed(123456)
```

Comment:
Can we test all possible combinations, i.e., `torch.randint(high, size)`, `torch.randint(low, high, size)` and the out variants? The above `test_rand` and `test_randn` are not the best examples... Sorry about those. 

Additionally, can you also check that the sampled values are within bounds?

Reply:
Any particular function I should use for checking bounds or simple combination of < > operators and summing up tensor?

Reply:
Anything that is reasonable to you is good :)